client/my-sites/woocommerce/landing-page.tsx

```diff
@@ -77,7 +77,7 @@ const WoopLandingPage: React.FunctionComponent< Props > = ( { siteId } ) => {
 				secondaryAction={ __( 'Learn more' ) }
 				secondaryActionURL="https://wordpress.com/support/introduction-to-woocommerce/"
 				secondaryActionTarget="_blank"
-				className="woop__landing-page-cta woocommerce_landing-page-empty-content"
+				className="landing-page__cta woocommerce_landing-page-empty-content"
```

Comment:
```suggestion
				className="landing-page-empty-content"
```

Reply:
Thanks! I think I made all the className changes. My linter insisted on `landing-page__` prefixes.

Reply:
Yea I think it just depends where you place them, if it was in woocommerce/index it would want `woocommerce__*`